types/fake-lodash-pure-js/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for fake-lodash-pure-js 1.0
+// Project: https://github.com/gxless/fake-lodash-pure-js
+// Definitions by: Xiang Gao <https://github.com/gxless>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+export function head<T>(array: T[]): T | undefined;
+export function tail<T>(array: T[]): T | undefined;
+export function isArray(input: any): boolean;
```

Comment:
is this OK to return boolean value, when function behind implementation seems to return only `T | undefined` for array like or just `undefined` for everything else?
https://github.com/gxless/fake-lodash-pure-js/blob/1c1aa16ddd0ebf6877049b55975d89888737f15d/src/services/head.js
https://github.com/gxless/fake-lodash-pure-js/blob/main/src/index.js#L2-L4

Reply:
fixed in https://github.com/gxless/fake-lodash-pure-js/blob/main/src/index.js#L2-L4
thanks for your review!